### DISCUSSION TITLE
Grant-PnPAzureADAppSitePermission

### DISCUSSION THREAD

**Discussion by Mig-00** [pnp/powershell#3765] [General]
Getting an error while running this cmdlet. Anyone knows when it will be working again? (Bug fix)

**Comment by nicole-ge** [comment]
Which error do you got?

**Comment by Mig-00** [comment]
Error : Grant-PnPAzureADAppSitePermission : {"error":{"code":"itemNotFound","message":"Item not
found","innerError":{"date":"2024-02-14T12:21:20","request-id":"PII","client-request-id":"PII"}}}

**Reply by nicole-ge** [reply]
Which powershell and pnp Version are you using?

**Reply by AndersRask** [reply]
What were the parameters for the cmdlet?
Also please post a stack trace of the error. An easy way to do this is to type `Get-PnPException` and paste the output here

**Comment by Mig-00** [comment]
@nicole-ge We are using 3.29.2101 version of PNP module.
